src/explore-education-statistics-frontend/src/modules/table-tool/components/TableTool.tsx

```diff
@@ -40,6 +44,18 @@ const TableToolFinalStep = ({
     setPermalinkId('');
   }, [tableHeaders, table]);
 
+  useEffect(() => {
+    const setGoToMethodologyLinkData = async () => {
```

Comment:
This is actually getting the release, this method and the associated variables be named as such as this adds confusion to any future developers as to what this actually is.